library/lib/components/dialogs/ods_alert_dialog.dart

```diff
@@ -0,0 +1,88 @@
+import 'package:flutter/material.dart';
+import 'package:ods_flutter/components/button/ods_filled_button.dart';
+import 'package:ods_flutter/guidelines/spacings.dart';
+
+class OdsAlertDialog extends StatefulWidget {
+  const OdsAlertDialog({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  State<OdsAlertDialog> createState() => _OdsAlertDialogState();
+
+  static void openDialog({
+    required BuildContext context,
+    required String text,
+    required String confirmButtonText,
+    Function()? onConfirmButtonClick,
+    String? titleText,
+    String? dismissButtonText,
```

Comment:
Can you create something (for example buttonDescription) to group buttonText and onButtonClick action. And use the element as optional